dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -3400,7 +3406,6 @@ RtpsUdpDataLink::RtpsWriter::gather_nack_replies_i(MetaSubmessageVec& meta_subme
           gaps.insert(seq);
         } else {
           // Non-directed gap.
-          consolidated_recipients.insert(addrs.begin(), addrs.end());
```

Comment:
Why is there no update to the recipients here?

Reply:
Gaps uses the bundling code to get addresses, not consolidated_recipients. I think this was a hold-over from pre-bundling days.